<launch>
	
	<node pkg="perception3d" type="data_publisher_joint_test" name="data_publisher_joint_test" respawn="false" output="screen" >
	
		<param name="joint_name" type="int" value="1"/>
		<param name="joint_velocity" type="int" value="10"/>
		<param name="joint_velocity2" type="int" value="5"/>
	</node>
	
</launch>
	
